src/app/core/http-services/joined-groups.service.ts

```diff
@@ -46,6 +52,8 @@ export class JoinedGroupsService {
       );
   }
 
-}
-
+  leave(id: string): Observable<LeaveGroupResponse> {
+    return this.http.delete<LeaveGroupResponse>(`${appConfig().apiUrl}/current-user/group-memberships/${id}`);
```

Comment:
For POST/PUT/DELETE, you have to use to map to `successData` instead... and here, probably map to just a boolean. Search for usage of `successData` at other places in the project...

Reply:
Fixed